### PR TITLE
Update dependencies

### LINKED ISSUE #3273: Support proto version 2
When using proto version 2 with required field in Armeria, try to support `gjson` or `gjson-web`, an error occurred when initializing `FramedGrpcService` and cannot be executed as below

```
2021-01-11 13:27:59.655 ERROR 6513 --- [           main] o.s.boot.SpringApplication               : Application run failed

org.springframework.context.ApplicationContextException: Unable to start reactive web server; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'com.linecorp.armeria.server.grpc.GrpcService#0': Unexpected exception during bean creation; nested exception is com.google.protobuf.UninitializedMessageException: Message missing required fields: message
        at org.springframework.boot.web.reactive.context.ReactiveWebServerApplicationContext.onRefresh(ReactiveWebServerApplicationContext.java:81) ~[spring-boot-2.4.0.jar:2.4.0]
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:582) ~[spring-context-5.3.1.jar:5.3.1]
        at org.springframework.boot.web.reactive.context.ReactiveWebServerApplicationContext.refresh(ReactiveWebServerApplicationContext.java:63) ~[spring-boot-2.4.0.jar:2.4.0]
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:767) ~[spring-boot-2.4.0.jar:2.4.0]
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:759) ~[spring-boot-2.4.0.jar:2.4.0]
        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:426) ~[spring-boot-2.4.0.jar:2.4.0]
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:326) ~[spring-boot-2.4.0.jar:2.4.0]
        at org.springframework.fu.kofu.KofuApplication.run(KofuApplication.kt:50) ~[spring-fu-kofu-0.5.0-patch-context.jar:na]
        at org.springframework.fu.kofu.KofuApplication.run$default(KofuApplication.kt:34) ~[spring-fu-kofu-0.5.0-patch-context.jar:na]
        at org.novonetworks.app.ApplicationKt.main(Application.kt:21) ~[main/:na]
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'com.linecorp.armeria.server.grpc.GrpcService#0': Unexpected exception during bean creation; nested exception is com.google.protobuf.UninitializedMessageException: Message missing required fields: message
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:544) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:233) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveNamedBean(DefaultListableBeanFactory.java:1235) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveBean(DefaultListableBeanFactory.java:494) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:349) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:342) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1179) ~[spring-context-5.3.1.jar:5.3.1]
        at org.novonetworks.app.ArmeriaServerConfigKt$armeriaServerConfig$1$1$$special$$inlined$bean$4.get(BeanDefinitionDsl.kt:1160) ~[main/:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.obtainFromSupplier(AbstractAutowireCapableBeanFactory.java:1233) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1175) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:571) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:531) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:213) ~[spring-beans-5.3.1.jar:5.3.1]
        at com.linecorp.armeria.spring.web.reactive.ArmeriaReactiveWebServerFactory.lambda$findBeans$6(ArmeriaReactiveWebServerFactory.java:302) ~[armeria-spring-boot2-webflux-autoconfigure-1.3.0.jar:na]
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) ~[na:na]
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) ~[na:na]
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[na:na]
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[na:na]
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[na:na]
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[na:na]
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[na:na]
        at com.linecorp.armeria.spring.web.reactive.ArmeriaReactiveWebServerFactory.findBeans(ArmeriaReactiveWebServerFactory.java:303) ~[armeria-spring-boot2-webflux-autoconfigure-1.3.0.jar:na]
        at com.linecorp.armeria.spring.web.reactive.ArmeriaReactiveWebServerFactory.getWebServer(ArmeriaReactiveWebServerFactory.java:207) ~[armeria-spring-boot2-webflux-autoconfigure-1.3.0.jar:na]
        at org.springframework.boot.web.reactive.context.WebServerManager.<init>(WebServerManager.java:49) ~[spring-boot-2.4.0.jar:2.4.0]
        at org.springframework.boot.web.reactive.context.ReactiveWebServerApplicationContext.createWebServer(ReactiveWebServerApplicationContext.java:93) ~[spring-boot-2.4.0.jar:2.4.0]
        at org.springframework.boot.web.reactive.context.ReactiveWebServerApplicationContext.onRefresh(ReactiveWebServerApplicationContext.java:78) ~[spring-boot-2.4.0.jar:2.4.0]
        ... 9 common frames omitted
Caused by: com.google.protobuf.UninitializedMessageException: Message missing required fields: message
        at com.google.protobuf.AbstractMessage$Builder.newUninitializedMessageException(AbstractMessage.java:477) ~[protobuf-java-3.12.0.jar:na]
        at greeting.GreetingProto$Greeting$Builder.build(GreetingProto.java:483) ~[main/:na]
        at greeting.GreetingProto$Greeting$Builder.build(GreetingProto.java:425) ~[main/:na]
        at org.curioswitch.common.protobuf.json.ProtoFieldInfo.valuePrototype(ProtoFieldInfo.java:133) ~[protobuf-jackson-1.1.0.jar:na]
        at org.curioswitch.common.protobuf.json.TypeSpecificMarshaller.buildOrFindMarshaller(TypeSpecificMarshaller.java:237) ~[protobuf-jackson-1.1.0.jar:na]
        at org.curioswitch.common.protobuf.json.TypeSpecificMarshaller.buildAndAdd(TypeSpecificMarshaller.java:160) ~[protobuf-jackson-1.1.0.jar:na]
        at org.curioswitch.common.protobuf.json.MessageMarshaller$Builder.build(MessageMarshaller.java:425) ~[protobuf-jackson-1.1.0.jar:na]
        at com.linecorp.armeria.common.grpc.GrpcJsonUtil.jsonMarshaller(GrpcJsonUtil.java:62) ~[armeria-grpc-1.3.0.jar:na]
        at com.linecorp.armeria.common.grpc.GrpcJsonMarshallerBuilder.build(GrpcJsonMarshallerBuilder.java:71) ~[armeria-grpc-1.3.0.jar:na]
        at com.linecorp.armeria.common.grpc.GrpcJsonMarshaller.of(GrpcJsonMarshaller.java:45) ~[armeria-grpc-1.3.0.jar:na]
        at com.linecorp.armeria.internal.shaded.guava.collect.CollectCollectors.lambda$toImmutableMap$1(CollectCollectors.java:61) ~[armeria-1.3.0.jar:na]
        at java.base/java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169) ~[na:na]
        at java.base/java.util.stream.DistinctOps$1$2.accept(DistinctOps.java:175) ~[na:na]
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) ~[na:na]
        at java.base/java.util.Collections$2.tryAdvance(Collections.java:4747) ~[na:na]
        at java.base/java.util.Collections$2.forEachRemaining(Collections.java:4755) ~[na:na]
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[na:na]
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[na:na]
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[na:na]
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[na:na]
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[na:na]
        at com.linecorp.armeria.server.grpc.FramedGrpcService.<init>(FramedGrpcService.java:131) ~[armeria-grpc-1.3.0.jar:na]
        at com.linecorp.armeria.server.grpc.GrpcServiceBuilder.build(GrpcServiceBuilder.java:584) ~[armeria-grpc-1.3.0.jar:na]
        at org.novonetworks.app.ArmeriaServerConfigKt.grpcService(ArmeriaServerConfig.kt:51) ~[main/:na]
        at org.novonetworks.app.ArmeriaServerConfigKt$armeriaServerConfig$1$1$$special$$inlined$bean$2.get(BeanDefinitionDsl.kt:1163) ~[main/:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.obtainFromSupplier(AbstractAutowireCapableBeanFactory.java:1233) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1175) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:571) ~[spring-beans-5.3.1.jar:5.3.1]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:531) ~[spring-beans-5.3.1.jar:5.3.1]
        ... 40 common frames omitted


FAILURE: Build failed with an exception.
```

### LINKED ISSUE #3286: Jetty JavaDoc broken link reported during build "configure" stage
Armeria build constantly print following messages to the output:

```
> Configure project :
Download https://www.eclipse.org/jetty/javadoc/current/package-list
Download failed: 404 Not Found
Download https://www.eclipse.org/jetty/javadoc/current/element-list
Download failed: 404 Not Found
```

Apparently `https://www.eclipse.org/jetty/javadoc/current/package-list` no longer exists.

### PR DESCRIPTION
**Pull request by ikhoon**
- Bouncy Castle 1.67 -> 1.68
- Brave 5.13.1 -> 5.13.3
- Caffiene 2.8.6 -> 2.8.8
- Dropwizard 1.3.25 -> 1.3.29
- Dropwizard 2.0.16 -> 2.0.18
- Dropwizard Metrics 4.1.15 -> 4.1.17
- fastutil 8.4.3 -> 8.4.4
- Guava 30.0 -> 30.1
- gRPC-Java 1.33.1 -> 1.35.0
- gRPC-Kotlin 0.2.1 -> 1.0.0
- Hibernate validator 6.1.6 -> 6.2.0
- java-jwt 3.11.0 -> 3.12.1
- Jetty 9.4.35 -> 9.4.36
- Micrometer 1.6.1 -> 1.6.3
- Netty 4.1.54 -> 4.1.58
  - tcnative 2.0.34 -> 2.0.36
  - io_uring 0.0.1 -> 0.0.3
- Protobuf Jackson 1.1.0 -> 1.2.0
- Reactor 3.4.0 -> 3.4.2
- RxJava 3.0.7 -> 3.0.9
- Scala 2.13.3 -> 2.13.4
- ScalaPB runtime 0.10.8 -> 0.10.10
- ScalaPB json4s 0.10.1 -> 0.10.3
- Spring 5.1.18 -> 5.1.20
- Spring 5.3.1 -> 5.3.3
- Spring Boot 2.1.17 -> 2.1.18
- Spring Boot 2.4.0 -> 2.4.2
- Tomcat 8.5.58 -> 8.5.61
- Tomcat 9.0.40 -> 9.0.41
- Examples
  - Dagger 2.30.1 -> 2.31.2
  - Kotlin allopen 1.4.20 -> 1.4.21-2
  - Reactor Kotlin extensions 1.1.0 -> 1.1.2
- Build
  - Checkstyle 8.38 -> 8.39
  - Error Prone 2.4.0 -> 2.5.1
  - Gradle 6.7.1 -> 6.8.1
- Test
  - Akka 2.6.10 -> 2.6.11
  - AssertJ 3.16.1 -> 3.17.1
  - Embedded Consul 2.1.4 -> 2.2.0
  - Eureka 1.10.10 -> 1.10.11
  - json-unit 2.22.0 -> 2.23.0
  - Mockito 3.6.28 -> 3.7.7
  - Prometheus client 0.9.0 -> 0.10.0
- Fixes #3273
- Fixes #3286

Co-authored-by: Trustin Lee <t@motd.kr>

### CODE REVIEW

**Review by trustin — APPROVED** [review]
That was a hard work, @ikhoon! :bow:

**Comment by ikhoon** [comment]
The following test fail depending on Netty version.
- Passed: 4.1.44 ~ 4.1.46
- Failed: 4.1.47 ~ 4.1.78
```
ServerTlsValidationTest > testPkcs12KeyStoreWithNullPassword() FAILED
    java.lang.AssertionError:
    Expecting code to raise a throwable.
        at com.linecorp.armeria.server.ServerTlsValidationTest.testPkcs12KeyStoreWithNullPassword(ServerTlsValidationTest.java:73)
```

Let me take a closer look tomorrow.

File: core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java [hunk 1]
```diff
@@ -1088,7 +1089,10 @@ private static SslContext validateSslContext(SslContext sslContext, boolean tlsA
             clientEngine.wrap(appBuf, packetBuf);
             appBuf.clear();
             packetBuf.flip();
-            serverEngine.unwrap(packetBuf, appBuf);
+            final SSLEngineResult result = serverEngine.unwrap(packetBuf, appBuf);
+            if (result.bytesConsumed() == 0) {
+                throw new IllegalStateException("failed to validate SSL/TLS configuration: " + sslContext);
+            }
```
Comment (ikhoon) [inline core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java:1095]: `ServerTlsValidationTest.testPkcs12KeyStoreWithNullPassword()` failure is caused by https://github.com/netty/netty/pull/10920 applied to Netty [4.1.57](https://netty.io/news/2021/01/12/4-1-57-Final.html).
An SSLException is not raised anymore on an invalid TLS configuration.

Comment (ikhoon) [reply core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java]: An SSLException raised while reading encrypted data is handled by `handleUnwrapException(...)`
https://github.com/netty/netty/blob/3353bc115217ccb292c95680da063212ae94d533/handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java#L1217-L1222
and returned an `SSLEngineResult` with `OK` status.
https://github.com/netty/netty/blob/3353bc115217ccb292c95680da063212ae94d533/handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java#L1335-L1337
So `result.bytesConsumed()` is used for checking correctness.

**Comment by trustin** [comment]
Here's the diff that fixes the TLS validation properly:

```diff
diff --git core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java
index 713157695..e6b1f73b8 100644
--- core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java
+++ core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java
@@ -72,6 +72,7 @@ import com.linecorp.armeria.server.logging.AccessLogWriter;
 import io.netty.buffer.ByteBufAllocator;
 import io.netty.handler.ssl.SslContext;
 import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.util.InsecureTrustManagerFactory;
 import io.netty.handler.ssl.util.SelfSignedCertificate;
 import io.netty.util.ReferenceCountUtil;
 
@@ -1021,7 +1022,7 @@ public final class VirtualHostBuilder {
 
             // Validate the built `SslContext`.
             if (sslContext != null) {
-                validateSslContext(sslContext, tlsAllowUnsafeCiphers);
+                validateSslContext(sslContext);
                 checkState(sslContext.isServer(), "sslContextBuilder built a client SSL context.");
             }
 
@@ -1065,7 +1066,7 @@ public final class VirtualHostBuilder {
      * key store password is not given to key store when {@link SslContext} was created using
      * {@link KeyManagerFactory}, the validation will fail and an {@link IllegalStateException} will be raised.
      */
-    private static SslContext validateSslContext(SslContext sslContext, boolean tlsAllowUnsafeCiphers) {
+    private static SslContext validateSslContext(SslContext sslContext) {
         if (!sslContext.isServer()) {
             throw new IllegalArgumentException("sslContext: " + sslContext + " (expected: server context)");
         }
@@ -1078,21 +1079,28 @@ public final class VirtualHostBuilder {
             serverEngine.setUseClientMode(false);
             serverEngine.setNeedClientAuth(false);
 
+            // Create a client-side engine with very permissive settings.
             final SslContext sslContextClient =
-                    buildSslContext(SslContextBuilder::forClient, tlsAllowUnsafeCiphers, ImmutableList.of());
+                    buildSslContext(() -> SslContextBuilder.forClient()
+                                                           .trustManager(InsecureTrustManagerFactory.INSTANCE),
+                                    true, ImmutableList.of());
             clientEngine = sslContextClient.newEngine(ByteBufAllocator.DEFAULT);
             clientEngine.setUseClientMode(true);
+            clientEngine.setEnabledProtocols(clientEngine.getSupportedProtocols());
+            clientEngine.setEnabledCipherSuites(clientEngine.getSupportedCipherSuites());
 
-            final ByteBuffer appBuf = ByteBuffer.allocate(clientEngine.getSession().getApplicationBufferSize());
             final ByteBuffer packetBuf = ByteBuffer.allocate(clientEngine.getSession().getPacketBufferSize());
 
-            clientEngine.wrap(appBuf, packetBuf);
-            appBuf.clear();
+            // Wrap an empty app buffer to initiate handshake.
+            wrap(clientEngine, packetBuf);
+
+            // Feed the handshake packet to the server engine.
             packetBuf.flip();
-            final SSLEngineResult result = serverEngine.unwrap(packetBuf, appBuf);
-            if (result.bytesConsumed() == 0) {
-                throw new IllegalStateException("failed to validate SSL/TLS configuration");
-            }
+            unwrap(serverEngine, packetBuf);
+
+            // See if the server has something to say.
+            packetBuf.clear();
+            wrap(serverEngine, packetBuf);
         } catch (SSLException e) {
             throw new IllegalStateException("failed to validate SSL/TLS configuration: " + e.getMessage(), e);
         } finally {
@@ -1103,6 +1111,41 @@ public final class VirtualHostBuilder {
         return sslContext;
     }
 
+    private static void unwrap(SSLEngine engine, ByteBuffer packetBuf) throws SSLException {
+        final ByteBuffer appBuf = ByteBuffer.allocate(engine.getSession().getApplicationBufferSize());
+        // Limit the number of wrap() calls to 8 times, to prevent a potential infinite loop.
+        // 8 is an arbitrary number, it can be any number greater than 2.
+        for (int i = 0; i < 8; i++) {
+            appBuf.clear();
+            final SSLEngineResult result = engine.unwrap(packetBuf, appBuf);
+            switch (result.getHandshakeStatus()) {
+                case NEED_UNWRAP:
+                    continue;
+                case NEED_TASK:
+                    engine.getDelegatedTask().run();
+                    continue;
+            }
+            break;
+        }
+    }
+
+    private static void wrap(SSLEngine sslEngine, ByteBuffer packetBuf) throws SSLException {
+        final ByteBuffer appBuf = ByteBuffer.allocate(0);
+        // Limit the number of wrap() calls to 8 times, to prevent a potential infinite loop.
+        // 8 is an arbitrary number, it can be any number greater than 2.
+        for (int i = 0; i < 8; i++) {
+            final SSLEngineResult result = sslEngine.wrap(appBuf, packetBuf);
+            switch (result.getHandshakeStatus()) {
+                case NEED_WRAP:
+                    continue;
+                case NEED_TASK:
+                    sslEngine.getDelegatedTask().run();
+                    continue;
+            }
+            break;
+        }
+    }
+
     @Override
     public String toString() {
         return MoreObjects.toStringHelper(this).omitNullValues()
diff --git core/src/test/java/com/linecorp/armeria/server/ServerTlsValidationTest.java core/src/test/java/com/linecorp/armeria/server/ServerTlsValidationTest.java
index 3378e22d3..e118c6628 100644
--- core/src/test/java/com/linecorp/armeria/server/ServerTlsValidationTest.java
+++ core/src/test/java/com/linecorp/armeria/server/ServerTlsValidationTest.java
@@ -21,6 +21,7 @@ import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import java.security.KeyStore;
 
 import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLHandshakeException;
 
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
@@ -54,6 +55,7 @@ class ServerTlsValidationTest {
                                        .tls(kmf)
                                        .build())
                 .isInstanceOf(IllegalStateException.class)
+                .hasCauseInstanceOf(SSLHandshakeException.class)
                 .hasMessageContaining("failed to validate SSL/TLS configuration");
     }
 
@@ -75,6 +77,7 @@ class ServerTlsValidationTest {
                                        .tls(kmf)
                                        .build())
                 .isInstanceOf(IllegalStateException.class)
+                .hasCauseInstanceOf(SSLHandshakeException.class)
                 .hasMessageContaining("failed to validate SSL/TLS configuration");
     }
 
```

**Review by minwoox — APPROVED** [review]
Thanks a lot!

**Pull request merged**